doc/answerfile.txt

```diff
@@ -198,6 +198,8 @@ Format of 'source' and 'driver-source'
         of the primary-disk element. Devices should be comma separated.
         The first device will be treated as the original, the second device will
         mirror the first.
+        The second device may be "missing" to proceed with a degraded RAID
```

Comment:
This is not exactly what the code is doing, the code is ignoring one `missing`, first or second